src/utils/FetchConfig.ts

```diff
@@ -1,20 +1,19 @@
 import { AxiosInstance as axios, GetFeatureFlag } from '@/utils'
 
 /**
- * Fetches config from environment and API.
- * @returns A promise to get & set session storage keys with appropriate values.
+ * Fetches config from environment and sets items in Session Storage or window object.
  */
-export async function FetchConfig (): Promise<any> {
+export function FetchConfig (): void {
   // get config from environment
   const origin = window.location.origin
   const processEnvVueAppPath: string = import.meta.env.VUE_APP_PATH
   const processEnvBaseUrl: string = import.meta.env.BASE_URL
 
   if (!origin || !processEnvVueAppPath || !processEnvBaseUrl) {
-    return Promise.reject(new Error('Missing environment variables'))
```

Comment:
I think this was left over from when this function made a network call to fetch the config.

The new code is cleaner!